src/analysis/convex/between.lean

```diff
@@ -0,0 +1,500 @@
+  wbtw R (f x) (f y) (f z) ↔ wbtw R x y z :=
+begin
+  refine ⟨λ h, _, λ h, h.map _⟩,
+  rwa [wbtw, ←affine_segment_image, hf.mem_set_image] at h
+end
+
+lemma function.injective.sbtw_map_iff {x y z : P} {f : P →ᵃ[R] P'} (hf : function.injective f) :
+  sbtw R (f x) (f y) (f z) ↔ sbtw R x y z :=
+by simp_rw [sbtw, hf.wbtw_map_iff, hf.ne_iff]
+
+@[simp] lemma affine_equiv.wbtw_map_iff {x y z : P} (f : P ≃ᵃ[R] P') :
+  wbtw R (f x) (f y) (f z) ↔ wbtw R x y z :=
+begin
+  refine function.injective.wbtw_map_iff (_ : function.injective f.to_affine_map),
+  exact f.injective
+end
+
+@[simp] lemma affine_equiv.sbtw_map_iff {x y z : P} (f : P ≃ᵃ[R] P') :
+  sbtw R (f x) (f y) (f z) ↔ sbtw R x y z :=
+begin
+  refine function.injective.sbtw_map_iff (_ : function.injective f.to_affine_map),
+  exact f.injective
+end
+
+omit V'
+
+@[simp] lemma wbtw_const_vadd_iff {x y z : P} (v : V) :
+  wbtw R (v +ᵥ x) (v +ᵥ y) (v +ᵥ z) ↔ wbtw R x y z :=
+mem_const_vadd_affine_segment _
+
+@[simp] lemma wbtw_vadd_const_iff {x y z : V} (p : P) :
+  wbtw R (x +ᵥ p) (y +ᵥ p) (z +ᵥ p) ↔ wbtw R x y z :=
+mem_vadd_const_affine_segment _
+
+@[simp] lemma wbtw_const_vsub_iff {x y z : P} (p : P) :
+  wbtw R (p -ᵥ x) (p -ᵥ y) (p -ᵥ z) ↔ wbtw R x y z :=
+mem_const_vsub_affine_segment _
+
+@[simp] lemma wbtw_vsub_const_iff {x y z : P} (p : P) :
+  wbtw R (x -ᵥ p) (y -ᵥ p) (z -ᵥ p) ↔ wbtw R x y z :=
+mem_vsub_const_affine_segment _
+
+@[simp] lemma sbtw_const_vadd_iff {x y z : P} (v : V) :
+  sbtw R (v +ᵥ x) (v +ᵥ y) (v +ᵥ z) ↔ sbtw R x y z :=
+by simp_rw [sbtw, wbtw_const_vadd_iff, (add_action.injective v).ne_iff]
+
+@[simp] lemma sbtw_vadd_const_iff {x y z : V} (p : P) :
+  sbtw R (x +ᵥ p) (y +ᵥ p) (z +ᵥ p) ↔ sbtw R x y z :=
+by simp_rw [sbtw, wbtw_vadd_const_iff, (vadd_right_injective p).ne_iff]
+
+@[simp] lemma sbtw_const_vsub_iff {x y z : P} (p : P) :
+  sbtw R (p -ᵥ x) (p -ᵥ y) (p -ᵥ z) ↔ sbtw R x y z :=
+by simp_rw [sbtw, wbtw_const_vsub_iff, (vsub_right_injective p).ne_iff]
+
+@[simp] lemma sbtw_vsub_const_iff {x y z : P} (p : P) :
+  sbtw R (x -ᵥ p) (y -ᵥ p) (z -ᵥ p) ↔ sbtw R x y z :=
+by simp_rw [sbtw, wbtw_vsub_const_iff, (vsub_left_injective p).ne_iff]
+
+lemma sbtw.wbtw {x y z : P} (h : sbtw R x y z) : wbtw R x y z :=
+h.1
+
+lemma sbtw.ne_left {x y z : P} (h : sbtw R x y z) : y ≠ x :=
+h.2.1
+
+lemma sbtw.left_ne {x y z : P} (h : sbtw R x y z) : x ≠ y :=
+h.2.1.symm
+
+lemma sbtw.ne_right {x y z : P} (h : sbtw R x y z) : y ≠ z :=
+h.2.2
+
+lemma sbtw.right_ne {x y z : P} (h : sbtw R x y z) : z ≠ y :=
+h.2.2.symm
+
+lemma sbtw.mem_image_Ioo {x y z : P} (h : sbtw R x y z) :
+  y ∈ line_map x z '' (set.Ioo (0 : R) 1) :=
+begin
+  rcases h with ⟨⟨t, ht, rfl⟩, hyx, hyz⟩,
+  rcases set.eq_endpoints_or_mem_Ioo_of_mem_Icc ht with rfl|rfl|ho,
+  { exfalso, simpa using hyx },
+  { exfalso, simpa using hyz },
+  { exact ⟨t, ho, rfl⟩ }
+end
+
+lemma wbtw_symm {x y z : P} : wbtw R x y z ↔ wbtw R z y x :=
```

Comment:
```suggestion
lemma wbtw_comm {x y z : P} : wbtw R x y z ↔ wbtw R z y x :=
```
I think the pattern in mathlib is to use `.symm` for the dot-notation and `_comm` for the `↔`.